excel-connector-test/src_test/com/axonivy/connector/excel/test/TestEntityClassCreator.java

```diff
@@ -0,0 +1,33 @@
+package com.axonivy.connector.excel.test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import com.axonivy.connector.excel.EntityClassReader;
+
+import ch.ivyteam.ivy.environment.IvyTest;
+
+@IvyTest
+public class TestEntityClassCreator {
+
+  @Test
+  void readToEntity(@TempDir Path dir) throws IOException {
+    Path path = dir.resolve("customers.xlsx");
+    try(InputStream is = TestEntityClassCreator.class.getResourceAsStream("sample.xlsx")) {
+      Files.copy(is, path, StandardCopyOption.REPLACE_EXISTING);
+    }
+
+    var reader = new EntityClassReader();
+    var entity = reader.getEntity(path.toFile().toString());
+    assertThat(entity).isNotNull();
+  }
```

Comment:
run me to create the 'customers' entity out of the given static sample.xlsx
![entityClassCreator](https://github.com/axonivy-market/excel-connector/assets/15085693/399de7fd-2426-45db-a20f-0e6ae16af1f7)